### PR TITLE
Fix resdk

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

**Comment by jkokosar** [comment]
@JureZmrzlikar How does omitting "output" influence the possible download of files from the data object _input_ fields? Was that ever even intended functionality - to also enable download of files from the input fields?

**Comment by JureZmrzlikar** [comment]
Currently (as far as i know) there is no possibility to download inputs. If one wishes to do so, it is best to check the corresponding sample (sample includes also the parent data object) and download from there. I do not know if we envision to overcome this in the future.

File: resdk/resolwe.py [hunk 1]
```diff
@@ -508,9 +509,15 @@ def get(self, uid):
 
         """
         if re.match('^[0-9]+$', str(uid)):  # iud is ID number:
```
Comment (dblenkus) [inline resdk/resolwe.py]: Use `isdigit` method: https://docs.python.org/3/library/stdtypes.html#str.isdigit

File: resdk/resolwe.py [hunk 2]
```diff
@@ -508,9 +509,15 @@ def get(self, uid):
...
+            if self.endpoint == 'presample':
+                return self.resource(id=uid, resolwe=self.resolwe, presample=True)
+            else:
+                return self.resource(id=uid, resolwe=self.resolwe)
         else:  # uid is slug
-            return self.resource(slug=uid, resolwe=self.resolwe)
+            if self.endpoint == 'presample':
+                return self.resource(slug=uid, resolwe=self.resolwe, presample=True)
+            else:
+                return self.resource(slug=uid, resolwe=self.resolwe)
```
Comment (dblenkus) [inline resdk/resolwe.py]: Write this like:

    sample_filter = {'id': uid} if uid.isdigit() else {'slug': uid}
    if self.endpoint == 'presample':
        sample_filter['presample'] = True
    return self.resource(resolwe=self.resolwe, **sample_filter)

**Comment by dblenkus** [comment]
@jkokosar if inputs are files, they are not actually stored on server. They must be processed by process and stored in `output` if you want to access them later. If inputs are Data objects, as @JureZmrzlikar pointed out, you can access that object and downloads it's outputs. 
If this is a common use-case (that someone needs outputs of an input object), you should present it and discuss it with @mstajdohar, so we can place it properly in our development cycle, so they don't interfere other tasks.

**Comment by dblenkus** [comment]
LGTM

**Comment by jkokosar** [comment]
@dblenkus I was thinking about Data objects, so the workflow @JureZmrzlikar and you proposed is sufficient.

**Pull request merged**